include/oneapi/dpl/pstl/hetero/dpcpp/execution_sycl_defs.h

```diff
@@ -50,14 +52,34 @@ class device_policy
     device_policy(const device_policy<OtherName>& other) : q(other.queue())
     {
     }
+
+    device_policy(const device_policy& other) : q(other.queue()) {}
+    device_policy(device_policy&& other) : q(other.queue()) {}
+    device_policy&
+    operator=(const device_policy& other)
+    {
+        q = other.queue();
+        return *this;
+    }
+    device_policy&
+    operator=(device_policy&& other)
+    {
+        q.swap(other.q);
```

Comment:
As form it's looks like not thread-safe operation too.

Reply:
Will change with the other assignment operator.

Reply:
Done.